mock.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Mock implementation for test runners.
+ *
+ * Example:
+ *
+ * ```js
+ * jest.mock('react-native-reanimated', () => require('react-native-reanimated/mock'));
+ * ```
+ */
+
+const React = require('react');
+const { View, Text, Image, ScrollView } = require('react-native');
+
+const NOOP = () => undefined;
+
+class Code extends React.Component {
+  static resolveNode = () => null;
```

Comment:
why is it needed?

Reply:
@osdnk there was this static method in the `Code` component, so I just added it in the mock

Reply:
https://github.com/kmagiera/react-native-reanimated/blob/master/src/core/AnimatedCode.js#L6